app/src/main/java/com/testtask/apptesttask/ui/AppActivity.kt

```diff
@@ -3,23 +3,18 @@ package com.testtask.apptesttask.ui
 import android.os.Bundle
 import com.arellomobile.mvp.MvpAppCompatActivity
 import com.arellomobile.mvp.MvpView
-import com.arellomobile.mvp.presenter.InjectPresenter
 import com.testtask.apptesttask.R
-import com.testtask.apptesttask.presentation.AppPresenter
 import com.testtask.apptesttask.ui.main.MainFragment
 
 class AppActivity : MvpAppCompatActivity(), MvpView {
 
-    @InjectPresenter
-    lateinit var presenter: AppPresenter
-
     override fun onCreate(savedInstanceState: Bundle?) {
-        setTheme(R.style.AppTheme)
         super.onCreate(savedInstanceState)
         setContentView(R.layout.layout_container)
 
-        if(savedInstanceState==null){
-            supportFragmentManager.beginTransaction().replace(R.id.container, MainFragment()).commit()
+        if (savedInstanceState == null) {
+            supportFragmentManager.beginTransaction().replace(R.id.container, MainFragment())
```

Comment:
Move `replace` to new line.

Reply:
Also one interested moment about this logic : in `Activity` you use `commit`, but in `Fragment` - ` commitNow`? Is it good?